src/app/home/DetailViewComponent.tsx

```diff
@@ -6,30 +6,47 @@ import { PlusCircleIcon } from '@patternfly/react-icons';
 import DetailViewItem from './components/DetailViewItem';
 import clusterOperations from '../cluster/duck/operations';
 import storageOperations from '../storage/duck/operations';
+import planOperations from '../plan/duck/operations';
 import AddClusterModal from '../cluster/components/AddClusterModal';
 import AddStorageModal from '../storage/components/AddStorageModal';
 import clusterSelectors from '../cluster/duck/selectors';
 import storageSelectors from '../storage/duck/selectors';
+import planSelectors from '../plan/duck/selectors';
+import { Creators as PlanCreators } from '../plan/duck/actions';
 import Wizard from '../plan/components/Wizard';
+import { css } from '@emotion/core';
+
 import {
   Button,
   ButtonVariant,
   DataList,
   InputGroup,
   TextInput,
 } from '@patternfly/react-core';
+
 interface IProps {
```

Comment:
This component needs to be refactored. Too many props - needs to be broken up into smaller components